integration/helpers/instance.go

```diff
@@ -963,6 +963,7 @@ func (i *TeleInstance) StartNodeAndProxy(t *testing.T, name string) (sshPort, we
 }
 
 // ProxyConfig is a set of configuration parameters for Proxy
+// TODO(tcsc): Add file descriptor slice to inject FDs into proxy process
```

Comment:
Is this a TODO for inclusion in this PR or just at a later date when we want to run all these tests in parallel.

Reply:
Later date. This PR was already pushing it on complexity (lots of big files moved around), if not raw size.